core/create/src/prompts/confirmation.ts

```diff
@@ -0,0 +1,42 @@
+import { styles } from '@dotcom-tool-kit/logger'
+import prompt from 'prompts'
+
+export interface ConfirmationParams {
+  deleteConfig: boolean
+  addEslintConfig: boolean
+  packagesToInstall: string[]
+  packagesToRemove: string[]
+  configFile: string
+}
+
+export default ({
+  deleteConfig,
+  addEslintConfig,
+  packagesToInstall,
+  packagesToRemove,
+  configFile
+}: ConfirmationParams): Promise<prompt.Answers<'confirm'>> =>
```

Comment:
**suggestion:** I think it would be nice to wrap this function in curly braces. It is one return statement but it takes up many lines and I automatically looked for the encapsulating curly braces to see where the function ends. I know it's fairly unlikely but I also think it would be useful to wrap in curly braces in case anything another function is added to this file to more clearly separate them 

```ts
}: ConfirmationParams): Promise<prompt.Answers<'confirm'>> => {
    return prompt({
    name: 'confirm',
    type: 'confirm',
    message: () => {
      return `so, we're gonna:
    ....
    ....
    })
}

```